### PR TITLE
 Multiplex download test for multiple content types 

### PR DESCRIPTION
**Pull request by mdellweg**

### CODE REVIEW

File: pulp_plugin_template/tests/functional/api/test_download_content.py [hunk 1]
```diff
@@ -86,18 +86,19 @@ def test_all(self):
...
+            utils.http_get(urljoin(PLUGIN_TEMPLATE_FIXTURE_URL, unit_path[0]))
+        ).hexdigest() for unit_path in unit_paths]
 
         # …and Pulp.
         client.response_handler = api.safe_handler
 
-        unit_url = cfg.get_hosts('api')[0].roles['api']['scheme']
-        unit_url += '://' + distribution['base_url'] + '/'
-        unit_url = urljoin(unit_url, unit_path)
+        unit_base_url = cfg.get_hosts('api')[0].roles['api']['scheme']
```
Comment (dralley) [inline pulp_plugin_template/tests/functional/api/test_download_content.py]: Copy this change https://github.com/pulp/pulp_file/pull/169/files

Comment (mdellweg) [reply pulp_plugin_template/tests/functional/api/test_download_content.py]: I copied it in the other PR. Let's rebase this afterwards.

**Comment by bmbouter** [comment]
@mdellweg this PR now has conflicts. What would you like to do with it?

**Comment by mdellweg** [comment]
@bmbouter rebased it, or rather reworked it...

**Pull request merged**